### DISCUSSION TITLE
Why can't I create an order automatically?

### DISCUSSION THREAD

**Discussion by HuJunnj** [openTCS/opentcs#125] [General]
Who can tell me why I can't create my order automatically?
<img width="779" alt="Screenshot 2024-01-29 225416" src="https://github.com/openTCS/opentcs/assets/75943306/b012312d-bd5d-4bd4-ba51-de319aeb0a05">

**Comment by swltr** [comment]
@HuJunnj You have already selected "Enable order generation", but with an "only once" trigger. If you want to create another set of orders, uncheck and recheck the "Enable order generation" checkbox.

**Comment by HuJunnj** [comment]
Hi thanks swltr, I have resolved this issue, must select the Create orders  according to definition. it will work normally.
